При повторной инициализации сервера происходит ошибка

# Команда для получения ошибки
```bash
npm run run
npm run run
```

# Лог ошибки
```
> unitables@1.0.0 run /project
> node src/server/main/main.js

fs.js:947
  handleErrorFromBinding(ctx);
  ^

Error: EEXIST: file already exists, mkdir 'data'
    at Object.mkdirSync (fs.js:947:3)
    at Server.configureDataRepo (file:///project/src/server/main/Server.js:59:8)
    at new Server (file:///project/src/server/main/Server.js:22:10)
    at file:///project/src/server/main/main.js:4:16
    at ModuleJob.run (internal/modules/esm/module_job.js:140:23)
    at async Loader.import (internal/modules/esm/loader.js:165:24)
    at async Object.loadESM (internal/process/esm_loader.js:68:5) {
  errno: -17,
  syscall: 'mkdir',
  code: 'EEXIST',
  path: 'data'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! unitables@1.0.0 run: `node src/server/main/main.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the unitables@1.0.0 run script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-08-11T10_06_46_524Z-debug.log

```

--- a/src/server/main/Server.js
+++ b/src/server/main/Server.js
@@ -56,7 +56,9 @@ export default class Server {
   }
 
   configureDataRepo() {
-    fs.mkdirSync(this.config.dataPath);
+    fs.mkdirSync(this.config.dataPath, {
+      recursive: true,
+    });
     const databasePath = `${this.config.dataPath}/${this.config.databaseName}`;
     const database = new Database(databasePath);
     this.dataRepo = new DataRepo(database, this.logger);

--- a/src/tests/server/main/Server.test.js
+++ b/src/tests/server/main/Server.test.js
@@ -3,10 +3,12 @@ import config from '../../../server/main/config.js';
 
 describe('Server', () => {
   describe('#main()', () => {
-    it('should run and close server', () => {
-      const server = new Server(config);
-      server.run();
-      server.close();
+    it('should create server twice successfully', () => {
+      for (let i = 0; i < 2; i += 1) {
+        const server = new Server(config);
+        server.run();
+        server.close();
+      }
     });
   });
 });

